mev_inspect/arbitrages.py

```diff
@@ -92,11 +92,21 @@ def _get_shortest_route(
     start_swap: Swap,
     end_swaps: List[Swap],
     all_swaps: List[Swap],
+    max_route_length: Optional[int] = None,
 ) -> Optional[List[Swap]]:
+    if len(end_swaps) == 0:
+        return None
+
+    if max_route_length is not None and max_route_length < 2:
+        return None
+
     for end_swap in end_swaps:
         if start_swap.token_out_address == end_swap.token_in_address:
             return [start_swap, end_swap]
 
+    if max_route_length is not None and max_route_length == 2:
```

Comment:
When do we expect this to be 2?

Reply:
when the shortest path is length 3

Reply:
Why should that return None?

Reply:
the parameter itself says "get the shortest route with a max length of 2"

beyond here there's no way to create a route that's shorter than 3, so it returns none